website/src/roadmap/python-sdk.md

```diff
@@ -22,7 +22,7 @@ limitations under the License.
 
 ## Python 3 Support
```

Comment:
We won't have Dataflow containers for 3.8 released with 2.16.0 for example, we won't have have tests until we add them, and some dependencies may have issues on 3.8 (e.g. dill).  Beam itself on direct runner may just work, but it would be more safe to say 3.5, 3.6, 3.7 as those are the classifiers we add on pypi.

Reply:
Ack. Let's go with "3.5, 3.6, 3.7"